index.html

```diff
@@ -4,20 +4,23 @@
     <title>Coding Student 2021</title>
     <meta name="author" content ="Melania Solis">
     <meta name="description" content="Personal Information">
+    <link rel='stylesheet' href='css/index.css'>
   </head>
 
   <body>
     <h2>Melania Solis</h2>
       <section>
+        <h2>About</h2>
         <ul>
-          <li><h2>About</h2></li>
-          <p>I'm originally from <a href="https://en.wikipedia.org/wiki/Nicaragua">Nicaragua</a>.  I'm learning javascript with Code the Dream</p>
+          <li><p>I'm originally from <a href="https://en.wikipedia.org/wiki/Nicaragua">Nicaragua</a>.  I'm learning javascript with Code the Dream</p></li>
```

Comment:
Having the <p> tags within your <li> doesn't hurt anything, but changes nothing about the appearance of your page versus having just the <li> tags.  Same thing in line 34.